### PR TITLE
Add Cassandra installation script

### PR DESCRIPTION
**Pull request by yliu0715**
Installation of Cassandra test on Ubuntu 19.04

### CODE REVIEW

File: scripts/lib/install_cassandra.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Cassandra Installation Script
+# Reference: http://cassandra.apache.org/download/
+#
+# Install Java 8 if not installed
+install_java_environment() {
+  command -v java >/dev/null 2>&1;
+  if [[ $? -eq 1 ]]; then
+    echo "Java not installed..."
+    echo "Installing java runtime..."
+    sudo apt-get -y install openjdk-8-jre
```
Comment (MicLi01) [inline scripts/lib/install_cassandra.sh]: I believe that if you install the jdk it already has the jre in it so you won't need this line.

**Comment by MicLi01** [comment]
Reading package lists... Done                                                                                                                                                                                                                                                 
W: GPG error: http://dl.bintray.com/apache/cassandra 311x InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A278B781FE4B2BDA
E: The repository 'http://www.apache.org/dist/cassandra/debian 311x InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Installing Cassandra...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package cassandra is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'cassandra' has no installation candidate
Modifying server hostname...
sed: -e expression #1, char 121: unterminated `s' command
Successfully installed the dependencies

is this suppose to happen?

**Comment by yliu0715** [comment]
> Reading package lists... Done
> W: GPG error: http://dl.bintray.com/apache/cassandra 311x InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A278B781FE4B2BDA
> E: The repository 'http://www.apache.org/dist/cassandra/debian 311x InRelease' is not signed.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.
> Installing Cassandra...
> Reading package lists... Done
> Building dependency tree
> Reading state information... Done
> Package cassandra is not available, but is referred to by another package.
> This may mean that the package is missing, has been obsoleted, or
> is only available from another source
> 
> E: Package 'cassandra' has no installation candidate
> Modifying server hostname...
> sed: -e expression #1, char 121: unterminated `s' command
> Successfully installed the dependencies
> 
> is this suppose to happen?
Did you run this with the latest commit?

**Comment by yliu0715** [comment]
@MicLi01 Did you run it with the latest commit?

**Comment by yliu0715** [comment]
@MicLi01 It was because I forgot a space in the if statement inside the install_curl function.

**Pull request merged**